src/devices/shyugj.ts

```diff
@@ -0,0 +1,22 @@
+import {battery, iasZoneAlarm} from '../lib/modernExtend';
+import {DefinitionWithExtend} from '../lib/types';
+
+const definitions: DefinitionWithExtend[] = [
+    {
+        zigbeeModel: ['DoorSensor-ZB3.0'],
+        model: 'DoorSensor-ZB3.0',
```

Comment:
Is there a better model number written on the device or product page?

Reply:
The only other identifier I have is an item number on the setup documentation, `S901D-ZG`, if that feels more useful?